packages/cli/package.json

```diff
@@ -78,16 +78,16 @@
     "all": true
   },
   "dependencies": {
-    "@igniteui/angular-templates": "~17.1.13112",
-    "@igniteui/cli-core": "~13.1.12",
+    "@igniteui/angular-templates": "~17.1.13113-beta.6",
+    "@igniteui/cli-core": "~13.1.13-beta.6",
```

Comment:
Are those supposed to be changed by this PR? We usually change versions is a singular version commit before tagging, no?

Reply:
It's needed when we tag beta versions from a feat branch/PR not from master

Reply:
Still, don't care for that to be merged in master. Could consider changing the release process if you'll keep tagging from branches and have the release action set the version from the tag or something